experiment-setup/bootstrap/provisioning/server-provisioning.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+check_command() {
+    if [ $? -ne 0 ]; then
+        echo "Error: $1 failed."
+        exit 1
+    fi
+}
+
+echo "Starting provisioning for Docker on Alpine Linux..."
+
+apk update
+check_command "apk update"
+
+echo "Installing Docker and openrc (manager)..."
+apk add --no-cache openrc docker
+check_command "Docker installation"
+
+echo "Installing Docker Compose..."
+apk add --no-cache docker-compose
+check_command "Docker Compose installation"
+
+echo "Enabling and starting Docker service ..."
+rc-update add docker default # or boot here...
+check_command "Install and set service"
+
+echo "Provisioning complete. Docker and Docker Compose are installed."
+
+echo "Starting Docker..."
+service docker start # if this guy don't work then we have a problem
+
+echo "Pulling the images!"
+
+DOCKER_IMAGES="dudustri/waect-rust-actix alpine:edge" #list the images here
```

Comment:
I guess you used it for testing. The image name for pulling the image from our DockerHub Account will be:
`simonharwick97822/rust-actix:latest` (for the latest image we deployed) for the `rust-actix` application. 
If we want to pull versions it would be:
`simonharwick97822/rust-actix:commit-sha-6cd7c26-202410301126`

Reply:
ok, cool, I will change it